assets/javascript/game.js

```diff
@@ -25,6 +25,9 @@ window.Game = React.createClass({
       }
 
       $.get('/game/' + this.props.gameID, (data) => {
+          if (this.state.game && data.created_at != this.state.game.created_at) {
+            this.setState({codemaster: false});
```

Comment:
Calling `setState` twice in this function seems fine because React batches `setState` calls into a single update for performance: https://facebook.github.io/react/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous